src/Microsoft.ML.Data/Transforms/KeyToVector.cs

```diff
@@ -573,7 +573,14 @@ private ValueGetter<VBuffer<float>> MakeGetterInd(DataViewRow input, int iinfo)
                                 if (key >= (uint)size)
                                     continue;
```

Comment:
It seems that we need to use `Indices` anyway, in case this condition is ever true, don't we?

Reply:
@codemzs , it seems @Ivanidzo4ka had a PR for this: #2681 that hasn't been checked in for some reason.